src/Entity/EntityMetaRelation.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\entity_meta_relation\Entity;
+
+use Drupal\Core\Entity\EntityChangedTrait;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Entity\RevisionableContentEntityBase;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\entity_meta_relation\EntityMetaRelationInterface;
+
+/**
+ * Defines the entity meta relation entity class.
+ *
+ * @ContentEntityType(
```

Comment:
The same remarks go here as for the EntityMeta entity type. Minus, perhaps, the translatability?